ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml

```diff
@@ -1142,6 +1142,31 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-upi
+- as: aws-ipi-amd-f7-netobserv
```

Comment:
@liangxia Shall we place the job in alphabet order?

Reply:
If job not sorted in alphanumeric order is the only issue,  it's not a merge blocker.
If there are other issues need to address in the PR, then we ask the author to sort the job, too, together with other changes.